### PR TITLE
Removing reveal from controls

### PR DESCRIPTION
**Pull request by beervoley**
## Description

 
This PR removes reveal animations from the controls (except CalendarView and RadioMenuFlyoutItem).
RadioMenuFlyoutItem is in a separate [PR](https://github.com/microsoft/microsoft-ui-xaml/pull/4317)

### CODE REVIEW

File: dev/TabView/TabView.xaml [hunk 1]
```diff
@@ -194,7 +194,6 @@
                                 Delay="50"
                                 FontFamily="{ThemeResource SymbolThemeFontFamily}"
                                 Interval="100"
-                                Style="{StaticResource RepeatButtonRevealStyle}"
```
Comment (ranjeshj) [inline dev/TabView/TabView.xaml:197]: what does this look like now ?

File: dev/TreeView/TreeViewItem.xaml [hunk 2]
```diff
@@ -8,7 +8,7 @@
     xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)">
     
     <Style TargetType="local:TreeViewItem"
-           BasedOn="{StaticResource ListViewItemRevealStyle}"
```
Comment (ranjeshj) [inline dev/TreeView/TreeViewItem.xaml:11]: >ListViewItemRevealStyle [](start = 36, length = 23)

we should test this out and see nothing is broken.

File: dev/TabView/TabView.xaml (hunk 1 above)
Comment (beervoley) [reply dev/TabView/TabView.xaml:197]: With the change:
image
Without the change:
image
TabView will be update soon:)

File: dev/TreeView/TreeViewItem.xaml (hunk 2 above)
Comment (beervoley) [reply dev/TreeView/TreeViewItem.xaml:11]: will do

File: dev/TabView/TabView.xaml (hunk 1 above)
Comment (ranjeshj) [reply dev/TabView/TabView.xaml:197]: @YuliKl @stmoy  Is this okay for now until we fix TabView?

Comment (karenbtlai) [reply dev/TabView/TabView.xaml:197]: What work needs to be done for TabView? Leon is investigating drop shadows for TabViews right now, and wondering if future visual updates will regress these fixes.

Comment (ranjeshj) [reply dev/TabView/TabView.xaml:197]: This seems okay for now since we don't have the finalized design for these arrows.

**Review by ranjeshj — APPROVED** [review]

**Pull request merged**